plugins/network-elements/juniper-contrail/src/test/java/org/apache/cloudstack/network/contrail/management/ManagementServerMock.java

```diff
@@ -86,6 +86,7 @@
 import com.cloud.vm.dao.UserVmDao;
 
 public class ManagementServerMock {
+    protected Logger logger = LogManager.getLogger(getClass());
```

Comment:
@DaanHoogland is this required, maybe we wanted to do changes at line # 223-229, # 271 - 275?

```
catch (AgentUnavailableException | OperationTimedoutException e) {
    logger.error(e);
}
```

Reply:
tnx, yes.

Reply:
not sure why this file went so wrong. I’ll revisit it completely.